server/liveblog/items/items.py

```diff
@@ -50,6 +50,16 @@ class ItemsResource(ArchiveResource):
         'item_type': {
             'type': 'string'
         },
+        'name': {
+            'type': 'string',
+            'minlength': 1,
+            'maxlength': 30
+        },
+        'content': {
```

Comment:
I moved those two fields (name and content) inside `client_modules` module so that they are comments specific now